app/scripts/lib/metrics.js

```diff
@@ -364,6 +367,27 @@ define(function (require, exports, module) {
       // The idenifier is kept to avoid updating all metrics queries.
       return 'screen.' + viewName;
     },
+    /**
+     * Log when an experiment is shown to the user
+     *
+     * @param {String} choice - type of experiment
+     * @param {String} group - the experiment group (treatment or control)
+     */
+
+    logExperiment: function (choice, group) {
```

Comment:
this new function needs a unit test, see example: https://github.com/mozilla/fxa-content-server/blob/master/app/tests/spec/lib/metrics.js#L552